src/csharp/Record/Playback.cs

```diff
@@ -0,0 +1,786 @@
+﻿//------------------------------------------------------------------------------
+// <copyright file="Playback.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// </copyright>
+//------------------------------------------------------------------------------
+using System;
+using System.Text;
+using Microsoft.Azure.Kinect.Sensor.Record.Exceptions;
+
+namespace Microsoft.Azure.Kinect.Sensor.Record
+{
+    /// <summary>
+    /// Respresents a file being used to playback data from an Azure Kinect device.
+    /// </summary>
+    public class Playback : IDisposable
+    {
+        // The native handle for this recording.
+        private readonly NativeMethods.k4a_playback_t handle;
+
+        // To detect redundant calls to Dispose
+        private bool disposedValue = false;
+
+        // Cached values
+        private Calibration? calibration = null;
+        private RecordConfiguration recordConfiguration = null;
+
+        private Playback(NativeMethods.k4a_playback_t handle)
+        {
+            this.handle = handle;
+        }
+
+        /// <summary>
+        /// Gets get the camera calibration for Azure Kinect device used during recording. The output struct is used as input to all transformation functions.
```

Comment:
There are many places in the file where you have "Gets get", you can remove the duplicate "get".